compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -516,7 +516,10 @@ def delete_endpoint(
 
         endpoint_id = Endpoint.get_endpoint_id(endpoint_dir)
         if endpoint_id is None:
-            log.warning(f"Endpoint <{ep_name}> could not be located")
+            log.warning(
+                f"Endpoint named <{ep_name}>'s configuration not found, it "
+                "might not have been initialized locally"
```

Comment:
Minor suggested changes:
```suggestion
                f"Configuration for endpoint <{ep_name}> could not be found, it "
                "might not have been initialized locally"
```

Reply:
Changed